outputs.tf

```diff
@@ -23,5 +23,5 @@ output "role_arn" {
 
 output "security_group_id" {
   description = "The ID of the security group."
-  value       = "${aws_security_group.vpc.*.id[0]}"
+  value       = "${element(aws_security_group.vpc.*.id, 0)}"
```

Comment:
Think you have to use the splat-syntax including `concat()`'ing with another empty list - `list("")` - for this to work as expected? See example here:
https://github.com/telia-oss/terraform-aws-acm-certificate/blob/master/outputs.tf#L6

Reply:
True, now it works for both cases.